shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/viewmodel/handler/feedingpoint/DefaultFeedingPointHandler.kt

```diff
@@ -115,7 +116,8 @@ class DefaultFeedingPointHandler @Inject constructor(
     }
 
     override fun CoroutineScope.fetchNearestFeedingPoint(userLocation: MapLocation) {
-        launch {
+        fetchByPriorityJob?.cancel()
```

Comment:
I meant cancel the coroutine inside right after selecting, this line doesn't prevent multiple selecting

Reply:
`take(1)` prevents this